Modules/IO/HDF5/test/itkHDF5ImageIOTest.cxx

```diff
@@ -30,12 +30,12 @@ HDF5ReadWriteTest(const char * fileName)
   std::cout << fileName << std::endl;
   int success(EXIT_SUCCESS);
   using ImageType = typename itk::Image<TPixel, 3>;
-  typename ImageType::RegionType    imageRegion;
-  typename ImageType::SizeType      size;
-  typename ImageType::IndexType     index;
-  typename ImageType::SpacingType   spacing;
-  typename ImageType::PointType     origin;
-  typename ImageType::DirectionType myDirection;
+  typename ImageType::RegionType  imageRegion;
+  typename ImageType::SizeType    size;
+  typename ImageType::IndexType   index;
+  typename ImageType::SpacingType spacing;
+  typename ImageType::PointType   origin;
```

Comment:
Obviously these five lines of code are only accidentally modified, because of automatic local variable declaration alignment (the clang-format option that spawned some discussion).